src/devices/infactory.c

```diff
@@ -92,17 +135,43 @@ static char *output_fields[] = {
         "channel",
         "temperature_F",
         "humidity",
+        "mic",
+        "battery",
+#ifdef DEBUG        
+        "rawmsg",
+#endif // DEBUG        
         NULL,
 };
 
+/*
+Analysis using Genuino (see http://gitlab.com/hp-uno, e.g. uno_log_433):
```

Comment:
Impressive analysis. We have a tool revdgst that can take apart 8-bit checksums. I don't think it can handle 4-bit CRCs yet though.

Reply:
I needed the difference message analysis, because for CRC computation two bits are at a different position than in the final output. At least I have learned a lot about CRCs in the process. Now I even know how to compute it out-of-order (if needed). :-)